src-js/screens/Admin/screens/Settings/index.js

```diff
@@ -26,6 +27,21 @@ import Tabs from 'modules/tabs';
     // @since 3.32.7, country validation is disabled.
     //CountryValidator('#wl-country-code', '#wl-site-language');
 
+    const YTAPiKeySelector = document.querySelector('#wordlift_videoobject_youtube_api_key');
+    const VimeoApiSelector = document.querySelector('#wordlift_videoobject_vimeo_api_key');
+    if ( YTAPiKeySelector !== null && VimeoApiSelector !== null ) {
```

Comment:
could this be simplified to [ ...selectors].filter(..not null).map( ..bind events) ?